test/connectivity/scripts/connectivityTest.sh

```diff
@@ -248,7 +251,8 @@ function process_args() {
                 '-metricsEndpointsCSV' ) saveNextArg=22;;
                 '-metricsScrapeFrequencyInSecs' ) saveNextArg=23;;
                 '-metricsUploadTarget' ) saveNextArg=24;;
-                '-hostPlatform' ) saveNextArg=25;;
+                '-deploymentFileName' ) saveNextArg=25;;
+                '-hostPlatform' ) saveNextArg=26;;
```

Comment:
I didn't see -storageAccountConnectionString parameter, please get the latest from master

Reply:
I'm not introducing that change in this PR. I'll do it on a different PR when I'm working on TRC.

Reply:
Pull from master. Done.